src/tmpfiles/tmpfiles.c

```diff
@@ -3435,9 +3429,10 @@ static int parse_line(
                         return log_syntax(NULL, LOG_ERR, fname, line, SYNTHETIC_ERRNO(EINVAL), "Credential name not valid: %s", i.argument);
 
                 r = read_credential(i.argument, &i.binary_argument, &i.binary_argument_size);
-                if (IN_SET(r, -ENXIO, -ENOENT)) {
+                if (ERRNO_IS_NOINFO(r)) {
```

Comment:
this drop the check abotu ENOENT?

Reply:
I dropped that hunk. As you said, it dropped the check for ENOENT, and I don't think it should be handled the same as the others.